Sources/BraintreePayPalNativeCheckout/BTPayPalNativeTokenizationClient.swift

```diff
@@ -29,15 +29,15 @@ class BTPayPalNativeTokenizationClient {
 
         apiClient.post(
             "v1/payment_methods/paypal_accounts",
-            parameters: tokenizationRequest.parameters(returnURL: returnURL)
+            parameters: tokenizationRequest.parameters(returnURL: approvalData?.returnURL?.absoluteString)
```

Comment:
I don't think we want to make this post request at all if the `returnURL` is nil.

We could either error out earlier in this function, or have this function require a non-nil string for `returnURL` like it did before, in addition to the optional `ApprovalData`.

Reply:
Either way, we'll need to add an error case to [BTPayPalNativeCheckoutError.swift](https://github.com/braintree/braintree_ios/blob/main/Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutError.swift) for a missing returnURL.

Prior, we were [force unwrapping the returnURL](https://github.com/braintree/braintree_ios/blob/9576e337797013bc2874f8b75f6e21989a21549c/Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift#L173), which we should have properly unwrapped and had an error case anyways.